### DISCUSSION TITLE
Any way of getting visibility into next release?

### DISCUSSION THREAD

**Discussion by cssinate** [color-js/color.js#336] [Q&A]
First off - a huge thanks to the creators and maintainers. This is such a cool project. I appreciate the work you do for this library, and in contributing to the CSS spec. We're in the midst of some really exciting times!

Is there a discussion or roadmap somewhere that determines when the next release will be created? I had created an issue (https://github.com/LeaVerou/color.js/issues/322) and it's been fixed. I'm just wondering if there's any public visibility into when that might be released.

Thanks again.

**Comment by LeaVerou** [comment]
Yes! #403 and https://github.com/color-js/color.js/milestone/2
